solr/test-framework/src/java/org/apache/solr/SolrTestCaseJ4.java

```diff
@@ -300,6 +301,7 @@ public static void setupTestCases() {
     System.setProperty("solr.v2RealPath", "true");
     System.setProperty("zookeeper.forceSync", "no");
     System.setProperty("jetty.testMode", "true");
+    System.setProperty("solr.zookeeper.connectionStrategy", TestConnectionStrategy.class.getName());
```

Comment:
Could we have a comment here re: how or where it is used?

Reply:
What about a comment on TestConnectionStrategy instead? See the changes for that in b386e144e2f

Reply:
I realized that I was missing some changes necessary for this to make sense when I tried to split it out of a different branch that I had. Please take another look and see ZkClientConnectionStrategy